Src/PerformanceCollector/Shared/Implementation/QuickPulse/QuickPulseConstants.cs

```diff
@@ -14,5 +14,20 @@ internal class QuickPulseConstants
         /// Transmission time header.
         /// </summary>
         internal const string XMsQpsTransmissionTimeHeaderName = "x-ms-qps-transmission-time";
+
+        /// <summary>
+        /// Instance name header.
+        /// </summary>
+        internal const string XMsQpsInstanceNameHeaderName = "x-ms-qps-instance-name";
+
+        /// <summary>
+        /// Stream id header.
+        /// </summary>
+        internal const string XMsQpsStreamIdHeaderName = "x-ms-qps-stream-id";
```

Comment:
A GUID that uniquely identifies a QuickPulse client. Even when there are multiple application instances hosted on the same machine, stream id will be unique for each of them.